Gems/PhysX/Code/Tests/CharacterGameplayControllerTests.cpp

```diff
@@ -107,6 +107,15 @@ namespace PhysX
         float m_timeStep = AzPhysics::SystemConfiguration::DefaultFixedTimestep;
     };
 
+    TEST_F(PhysXDefaultWorldTest, CharacterGameplayController_GravitySets)
+    {
+        float expectedGravityMultiplier = 2.5f;
```

Comment:
`expectedGravityMultiplier` can be const.

````
const float expectedGravityMultiplier = 2.5f;
````

Reply:
Added